### DISCUSSION TITLE
CS:APP 第三章学习笔记

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#15] [Comments]
# CS:APP 第三章学习笔记

CS:APP 第三章“Machine-Level Representation of Programs”的学习笔记。
这章的主要内容为汇编（machine-level programming）。

https://ouuan.moe/post/2022/09/csapp-3

**Comment by ouuan** [comment]
这些内容我自学了一周，而上课讲了五周（还没讲全），但我在这五周几乎没能有时间继续往后学（
